### PR TITLE
chore(art-quiz): clean up art-quiz post palette update

### PR DESCRIPTION
**Pull request by laurabeth**
The type of this PR is: **CHORE**

### Description

- [x] update palette version
- [x] remove redundant grid components
- [x] fix column fractions

### CODE REVIEW

**Review by dzucconi — APPROVED** [review]
👍 — I still think this could be further simplified with a simple flexbox layout.

File: yarn.lock [hunk 1]
```diff
@@ -87,7 +87,25 @@
...
+    luxon "^1.15"
+    proportional-scale "^4.0.0"
+    react-lazy-load-image-component "1.5.0"
+    react-remove-scroll "2.5.0"
+    styled-system "^5.1.5"
+    trunc-html "^1.1.2"
+    use-cursor "^1.2.3"
+    use-keyboard-list-navigation "2.4.2"
+
+"@artsy/palette@^24.4.0":
```
Comment (dzucconi) [inline yarn.lock:108]: Actually

Comment (dzucconi) [reply yarn.lock:108]: Why isn't this deduped.

File: package.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
     "@artsy/express-reloadable": "^1.7.0",
     "@artsy/fresnel": "3.5.0",
     "@artsy/multienv": "^1.2.0",
-    "@artsy/palette": "24.4.0",
+    "@artsy/palette": "24.5.0",
```
Comment (dzucconi) [inline package.json:82]: You need to update charts simultaneously, I believe.

**Review by dzucconi — CHANGES_REQUESTED** [review]
Update charts as well

File: package.json [hunk 3]
```diff
@@ -79,8 +79,8 @@
     "@artsy/express-reloadable": "^1.7.0",
     "@artsy/fresnel": "3.5.0",
     "@artsy/multienv": "^1.2.0",
-    "@artsy/palette": "24.4.0",
-    "@artsy/palette-charts": "23.4.0",
+    "@artsy/palette": "24.5.0",
+    "@artsy/palette-charts": "23.5.0",
```
Comment (dzucconi) [inline package.json:83]: Did you `yarn install` I'm not seeing the lock file update?

**Comment by desaijay26** [comment]
Hi @laurabeth , @dzucconi   - can you please provide the set-up guidelines for this project,  I think the documentation has quite half-baked details. Would like to learn and contribute back to the project. Or can you please redirect me to the SPOC of this project?

**Comment by dzucconi** [comment]
You can completely remove the package updates and just rebase with main now.

**Review by dzucconi — DISMISSED** [review]

File: yarn.lock [hunk 4]
```diff
@@ -87,7 +87,7 @@
   resolved "https://registry.yarnpkg.com/@artsy/palette-tokens/-/palette-tokens-4.0.0.tgz#ee0d157ad73ca0e2e252f896a21fba798b20d4d9"
   integrity sha512-Os2fimLkHgxWLZZ8xRPWm4t3mBTMMBsKO+Iqgn2d44VjhcN7w9XxAp8VK8sRSLJmcrajnTczZZtFy273lYeqaA==
 
-"@artsy/palette@24.5.1":
+"@artsy/palette@24.5.1", "@artsy/palette@^24.5.1":
```
Comment (laurabeth) [inline yarn.lock:90]: @dzucconi Still seeing this dupe here. Any ideas why?

Comment (dzucconi) [reply yarn.lock]: Actually that was it being deduped! https://github.com/artsy/force/pull/11201 wasn't merged. Anyway; all good now.

**Comment by dzucconi** [comment]
Wondering what a good way around that dupe problem is — I wonder if we can specify that palette and palette-charts should be updated within the same PR in dependabot.

**Comment by dzucconi** [comment]
I guess we could also simply make palette a peer dep in palette-charts!

**Pull request merged**